Oracle/shrine_AdapterMappings.sql

```diff
@@ -14,36 +14,74 @@ drop table SHRINE_ONT.temp_shrine_mapping;
 CREATE TABLE SHRINE_ONT.temp_shrine_mapping
    AS (
        -- diagnosis icd9
-        select '\\SHRINE'|| so.c_fullname scihls_path,
-               '\\i2b2_Diagnoses' || ht.c_fullname heron_path 
+        select
+        	'\\SHRINE'|| so.c_fullname scihls_path
+            ,'\\i2b2_Diagnoses' || ht.c_fullname heron_path 
```

Comment:
These are just whitespace changes, right? I don't see how they relate to procedures. Better do this sort of clean up in its own commit.

Reply:
ok I ll do that.